### DISCUSSION TITLE
Why are my contributions not showing up on my profile?

### DISCUSSION THREAD

**Discussion by sentamil99** [community/community#39743] [New to GitHub]
Why are my contributions not showing up on my profile?

**Comment by SeanWhelan117** [comment]
Are your repositories private? Others will only see the contributions you made to public repos

**Comment by Preetiraj3697** [comment]
1. Your local Git commit email isn't connected to your account.
Commits must be made with an email address that is connected to your account on GitHub.com, or the GitHub-provided noreply email address provided to you in your email settings, in order to appear on your contributions graph. 
2. Your repositories maybe private?

**Reply by igorovisk** [reply]
I have the same issue, although my email is linked to my accoount :(

**Comment by DenverCoder1** [comment]
> Commits will appear on your contributions graph if they meet all of the following conditions:
> 
> * The email address used for the commits is associated with your account on GitHub.com.
> * The commits were made in a standalone repository, not a fork.
> * The commits were made:
>   * In the repository's default branch
>   * In the gh-pages branch (for repositories with project sites)
> For more information on project sites, see "About GitHub Pages."

[Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)
